x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/use_alert_suppression.test.tsx

```diff
@@ -16,51 +16,15 @@ describe('useAlertSuppression', () => {
       .mockReturnValue(false);
   });
 
-  it('should return isSuppressionEnabled false if rule Type exists in SUPPRESSIBLE_ALERT_RULES and Feature Flag is disabled', () => {
-    const { result } = renderHook(() => useAlertSuppression('new_terms'));
-
-    expect(result.current.isSuppressionEnabled).toBe(false);
-  });
-
-  it('should return isSuppressionEnabled true if rule Type exists in SUPPRESSIBLE_ALERT_RULES and Feature Flag is enabled', () => {
-    jest
-      .spyOn(useIsExperimentalFeatureEnabledMock, 'useIsExperimentalFeatureEnabled')
-      .mockReset()
-      .mockReturnValue(true);
-    const { result } = renderHook(() => useAlertSuppression('new_terms'));
-
-    expect(result.current.isSuppressionEnabled).toBe(true);
-  });
-
-  it('should return the correct isSuppressionEnabled value fot threat_match rule type', () => {
-    const { result } = renderHook(() => useAlertSuppression('threat_match'));
-
-    expect(result.current.isSuppressionEnabled).toBe(true);
-  });
-
-  describe('Eql Rule', () => {
-    it('should return the correct isSuppressionEnabled value if rule Type exists in SUPPRESSIBLE_ALERT_RULES and Feature Flag is enabled', () => {
-      jest
-        .spyOn(useIsExperimentalFeatureEnabledMock, 'useIsExperimentalFeatureEnabled')
-        .mockReset()
-        .mockReturnValue(true);
-      const { result } = renderHook(() => useAlertSuppression('eql'));
-
-      expect(result.current.isSuppressionEnabled).toBe(true);
-    });
-
-    it('should return the correct isSuppressionEnabled value if rule Type exists in SUPPRESSIBLE_ALERT_RULES and Feature Flag is disabled', () => {
-      const { result } = renderHook(() => useAlertSuppression('eql'));
-
-      expect(result.current.isSuppressionEnabled).toBe(false);
-    });
-  });
-
-  it('should return the correct isSuppressionEnabled value if rule Type exists in SUPPRESSIBLE_ALERT_RULES', () => {
-    const { result } = renderHook(() => useAlertSuppression('query'));
-
-    expect(result.current.isSuppressionEnabled).toBe(true);
-  });
+  (['new_terms', 'threat_match', 'saved_query', 'query', 'threshold'] as Type[]).forEach(
```

Comment:
Should this just reference `SUPPRESSIBLE_ALERT_RULES` ?

Reply:
if we just traverse `SUPPRESSIBLE_ALERT_RULES`, tests would always be true, even if wrong rule type will be put in `SUPPRESSIBLE_ALERT_RULES` config. 
This way we able to control and explicitly test which rule types we expect to be suppressible.

also, when we introduce new rule type to suppression that would be behind FF, that would require to be changed to filter out new rule type under FF from `SUPPRESSIBLE_ALERT_RULES`